pygeos/constructive.py

```diff
@@ -438,6 +441,36 @@ def point_on_surface(geometry, **kwargs):
     """
     return lib.point_on_surface(geometry, **kwargs)
 
+
+@requires_geos("3.7.0")
+@multithreading_enabled
+def reverse(geometry, **kwargs):
+    """Returns a copy of the Geometry with the order of coordinates reversed.  If the
+    Geometry is a polygon with interior rings, the interior rings are also reversed.
+
```

Comment:
```suggestion
    """Returns a copy of the Geometry with the order of coordinates reversed.

    If the Geometry is a polygon with interior rings, the interior rings are also
    reversed.
```

(have the first one-liner of the docstring be the main summary. The inner rings note is already more a detail I think)